src/Microsoft.VisualStudio.ProjectSystem.Managed/Telemetry/ILanguageServiceTelemetryService.cs

```diff
@@ -0,0 +1,46 @@
+﻿// Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE.md file in the project root for more information.
+
+using Microsoft.VisualStudio.Composition;
+using Microsoft.VisualStudio.ProjectSystem;
+using Microsoft.VisualStudio.ProjectSystem.LanguageServices;
+
+namespace Microsoft.VisualStudio.Telemetry
+{
+    [ProjectSystemContract(ProjectSystemContractScope.Global, ProjectSystemContractProvider.Private, Cardinality = ImportCardinality.ExactlyOne)]
+    internal interface ILanguageServiceTelemetryService
```

Comment:
I don't know the convention here, but would `ILanguageServiceTelemetryEvents` be a more accurate name? I don't know if this interface itself represents a 'Service`.

Reply:
Out of curiosity, what do you consider a service? Would this apply to the existing ITelemetryService as well?

---
In reply to: [567168061](https://github.com/dotnet/project-system/pull/6880#discussion_r567168061) [](ancestors = 567168061)